nohwookjin/src/components/Comment.tsx

```diff
@@ -0,0 +1,138 @@
+import { ChangeEvent, FormEvent, KeyboardEvent, useState } from "react";
+import styled from "styled-components";
+import profile from "../../public/images/abstract-user-flat-4.png";
+import Image from "next/image";
+
+export interface IComment {
+  id: number;
+  name: string;
+  comment: string;
+}
+
+const Comment = () => {
+  const [comment, setComment] = useState<IComment[]>([]);
+  const [currentText, setCurrentText] = useState("");
+  const [time, setTime] = useState<string>("");
+
+  const commentTime = () => {
+    const date = new Date();
+    const year = String(date.getUTCFullYear());
+    const month = String(date.getUTCMonth() + 1);
+    const day = String(date.getUTCDate());
+    setTime(`${year}-${month}-${day}`);
+  };
+
+  const onSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (currentText.length < 1) return alert("댓글을 입력해주세요");
+    setComment((prev) => [
+      ...prev,
+      { id: prev.length + 1, name: "익명", comment: currentText },
+    ]);
+    setCurrentText("");
+    commentTime();
```

Comment:
```suggestion
    commentTime("");
```
이렇게 초기화를 해주는 게 좋을 거 같아요!